cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -5085,6 +5085,14 @@
     </description>
   </property>
 
+  <property>
+    <name>feature.pipeline.and.triggers.enabled</name>
```

Comment:
Sorry this is still confusing - can it be `feature.pipeline.triggers.conditional.and.enabled`? that way someone won't be able to confuse the `and` as part of the description and triggers is towards the front providing more context.

Reply:
Yeah I have to say this `and` is easy to confuse with. I'll update